database/schema/08-gov.sql

```diff
@@ -29,7 +29,7 @@ CREATE TABLE proposal_deposit
     amount            COIN[],
     timestamp         TIMESTAMP,
     height            BIGINT  NOT NULL,
-    CONSTRAINT unique_deposit UNIQUE (proposal_id, depositor_address)
+    CONSTRAINT unique_deposit UNIQUE (proposal_id, depositor_address, amount)
```

Comment:
Does it mean in a proposal it is indeed not possible to have multiple deposits with the same amount from the same depositor address? Or this change is simply taking advantage of indexing? @@

Reply:
I added this because sometimes the same address will pay the deposit in more than one txs (eg first deposit 100 DARIC, and later deposit 400 DARIC in a different tx) and right now only the latest deposit will be stored for given address inside the `proposal_deposit` table so by updating this constraint we can store all deposits correctly

Reply:
Also, I just updated it and added timestamp to the `unique_deposit` constraint to handle the edge cases where address can pay the same amount of deposit twice in different txs (eg. deposit 100 DARIC first, later deposit 100 DARIC again)

Reply:
In this case can we also store the `txHash` and use it as a unique constraint? 🤔 It sounds more like a unique identifier for a proposal deposit instead of relying on timestamp and amount?

Or if txn timestamp must be unique across deposits and txns, we can remove `amount` from the unique constraint?

Reply:
Ok I added `transaction_hash` to `proposal_deposit` table and updated `unique_deposit` constraint. I didn't create the reference to message table because genesis deposits doesn't contain tx hash details, so proposal deposits stored from genesis file will have empty string inside `transaction_hash` column